iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/pipe/PipeConsensusServerImpl.java

```diff
@@ -276,6 +276,10 @@ public synchronized void checkConsensusPipe(Map<ConsensusPipeName, PipeStatus> e
                   consensusGroupId,
                   existedName,
                   expectedStatus);
+              if (expectedStatus.equals(PipeStatus.RUNNING)) {
+                // Do nothing. Because Pipe framework's metaSync will do that.
```

Comment:
If this pipe already exists on configNode, pipe's meta sync can ensure correctness. Can we assure that the pipe is already known by configNode here?

Reply:
yes, we can